### PR TITLE
added tests

### LINKED ISSUE #2: Write tests

### PR DESCRIPTION
**Pull request by gabrielascalon**
closes #2

### CODE REVIEW

File: blog/tests.py [hunk 1]
```diff
@@ -1,3 +1,57 @@
...
+        url = reverse('post_detail', kwargs={'pk': '1234'})
+        response = self.client.get(url)
+        self.assertTrue(response.status_code == 404)
+
+    def test_post_detail_draft_not_logged_in(self):
+        post = Post(author=self.user,
+                    title = 'My title for draft not logged in test',
+                    text='test text')
+        post.save()
+        url = reverse('post_detail', kwargs = {'pk': post.pk})
```
Comment (Marcelo-Theodoro) [inline blog/tests.py]: We should not use spaces around the equal sign in the kwargs.

File: blog/tests.py [hunk 2]
```diff
@@ -1,3 +1,57 @@
...
+        response = self.client.get(url)
+        self.assertTrue(response.status_code == 404)
+
+    def test_post_detail_draft_logged_in(self):
+        post = Post(author=self.user,
+                    title = 'My title for draft in logged in test',
```
Comment (Marcelo-Theodoro) [inline blog/tests.py]: Same here.

File: blog/tests.py [hunk 3]
```diff
@@ -1,3 +1,57 @@
...
+                    text='test text')
+        post.save()
+        login = self.client.force_login(self.user)
+        url = reverse('post_detail', kwargs = {'pk': post.pk})
```
Comment (Marcelo-Theodoro) [inline blog/tests.py]: same...

**Review by Marcelo-Theodoro — CHANGES_REQUESTED** [review]
plz, fix it.

**Comment by gabrielascalon** [comment]
@Marcelo-Theodoro fixed

**Review by Marcelo-Theodoro — APPROVED** [review]

**Pull request merged**